### PR TITLE
Multi meters and Measurements Lab Doc

### PR DESCRIPTION
**Pull request by thcy07**

### CODE REVIEW

File: _labs/Multimeters and Measurements.md
Comment (jgoeders) [inline _labs/Multimeters and Measurements.md:1]: I rather not have spaces in our filenames.  Could you rename this to `multimeters_and_measures.md`?  I know you didn't create the filename, but we might as well fix it now.

File: assets/Multimeters and Measurements/Ohm's Law.png
Comment (jgoeders) [inline assets/Multimeters and Measurements/Ohm's Law.png:1]: Could you rename the files to not have spaces?  Thanks!

**Review by jgoeders — CHANGES_REQUESTED** [review]

File: _layouts/default.html [hunk 1]
```diff
@@ -28,5 +28,13 @@
...
-
+<script type="text/x-mathjax-config">
+  MathJax.Hub.Config({
+    tex2jax: {
+      inlineMath: [['$','$'], ['\\(','\\)']],
+      processEscapes: true
+    }
+  });
+  </script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.0/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
```
Comment (jgoeders) [inline _layouts/default.html]: Could you move this into scripts.html instead?  You will see it is included above.

**Review by jgoeders — CHANGES_REQUESTED** [review]

File: _labs/Multimeters and Measurements.md
Comment (thcy07) [reply _labs/Multimeters and Measurements.md:1]: Should I do that for all the files?

Comment (jgoeders) [reply _labs/Multimeters and Measurements.md:1]: Just the files in this pull request is fine.

File: _layouts/default.html (hunk 1 above)
Comment (thcy07) [reply _layouts/default.html]: There is no html file with that name, would you like me to create one?

Comment (jgoeders) [reply _layouts/default.html]: It's here: https://github.com/byu-cpe/ecen192/blob/main/_includes/scripts.html

Comment (thcy07) [reply _layouts/default.html]: Done. Thank You!

**Comment by thcy07** [comment]
Also since my lab is merged with the Intro to Electronics lab do we still want to keep that file?

**Comment by jgoeders** [comment]
If we aren't doing the lab, you can remove it.

**Pull request merged**